sycl/doc/design/CommandGraph.md

```diff
@@ -373,6 +375,8 @@ adapter where there is matching support for each function in the list.
 | urCommandBufferAppendMemBufferWriteRectExp |  | No |
 | urCommandBufferAppendMemBufferReadRectExp |  | No |
 | urCommandBufferAppendMemBufferFillExp | clCommandFillBufferKHR | Yes |
+| urCommandBufferAppendUSMPrefetchExp |  | No |
```

Comment:
This PR adds support to urCommandBufferAppendUSMPrefetchExp so shouldn't this be marked as supported?

etc

Reply:
Ignore the above, I read it correctly now, this table is only for opencl support.